aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -249,6 +255,84 @@ module aptos_framework::account {
         account_resource.authentication_key = new_auth_key;
     }
 
+    fun verify_key_rotation_signature(scheme: u8, public_key_bytes: vector<u8>, signature: vector<u8>, challenge: &RotationProofChallenge) : vector<u8> {
```

Comment:
should we call this function `get_auth_key_from_xxxx`? 
`verify_xxx` sounds like it should return a boolean value

Reply:
good point! updated the function name